### PR TITLE
KAFKA-8183: Add retries to WorkerUtils#verifyTopics

### PR DESCRIPTION
**Pull request by stanislavkozlovski**
https://issues.apache.org/jira/browse/KAFKA-8183

### CODE REVIEW

File: tools/src/main/java/org/apache/kafka/trogdor/common/WorkerUtils.java [hunk 1]
```diff
@@ -248,9 +248,25 @@ static void createTopics(
...
-        Map<String, TopicDescription> topicDescriptionMap = topicsResult.all().get();
+
+        Map<String, TopicDescription> topicDescriptionMap;
+        int retries = 0;
+        while (true) {
+            try {
+                DescribeTopicsResult topicsResult = adminClient.describeTopics(
+                        topicsToVerify, new DescribeTopicsOptions().timeoutMs(ADMIN_REQUEST_TIMEOUT));
+                topicDescriptionMap = topicsResult.all().get();
+                break;
```
Comment (cmccabe) [inline tools/src/main/java/org/apache/kafka/trogdor/common/WorkerUtils.java]: Suggest just using "return" here.  Then you can have the loop be a regular for loop.

**Comment by cmccabe** [comment]
LGTM

**Comment by stanislavkozlovski** [comment]
JDK 8 build passed successfully

**Comment by cmccabe** [comment]
Failing test is `org.apache.kafka.streams.internals.ApiUtilsTest#shouldThrowNullPointerExceptionForNullDuration`, which is not related.

**Pull request merged**